packages/ckeditor5-font/tests/ui/colorui.js

```diff
@@ -535,4 +535,61 @@ describe( 'ColorUI', () => {
 			expect( dropdown.colorSelectorView.colorPickerView ).to.be.undefined;
 		} );
 	} );
+
+	// Issue: https://github.com/ckeditor/ckeditor5/issues/15580
+	// For simplicity we create editor with two same buttons in toolbar
+	// instead overcomplicating stuff with ballon toolbar.
+	describe( 'toolabar with two same instance of testColor', () => {
+		let editor, element;
+
+		beforeEach( () => {
+			element = document.createElement( 'div' );
+			document.body.appendChild( element );
+
+			return ClassicTestEditor
+				.create( element, {
+					plugins: [ Paragraph, TestColorPlugin, Undo ],
+					testColor: testColorConfig,
+					toolbar: [
+						'testColor',
+						'testColor'
```

Comment:
This is useless if in the test you create a new dropdown with `dropdown = editor.ui.componentFactory.create( 'testColor' );`. If you want to create a dropdown this way, there is no need of adding these toolbar buttons (and maybe no need of creating a separate `describe` block, but it's up to you). Check the next comment about how to handle a case with 2 dropdowns created manually.